src/docs/running-tests.md

```diff
@@ -296,6 +296,24 @@ on_finish:
       find "$APPVEYOR_BUILD_FOLDER" -type f -name 'TEST*.xml' -print0 | xargs -0 -I '{}' curl -F 'file=@{}' "https://ci.appveyor.com/api/testresults/junit/$APPVEYOR_JOB_ID"
 ```
 
+### Transform CTest results to JUnit
+
+If using CTest you can still upload your xml test results by transforming them into JUnit compatible results. See the example script below.
```

Comment:
I would add something like "thanks to [this](https://github.com/rpavlik/jenkins-ctest-plugin/blob/master/ctest-to-junit.xsl) open source template" to clearly refer to the original code. Other than that looks great.